test/e2e/a11y.test.ts

```diff
@@ -59,7 +59,7 @@ describe('Checking examples accessibility', function () {
           await page.waitForVisible(tableRowSelector, true, 30000);
         } else if (pagePath.match(/form.html$/)) {
           // ace is loaded asynchronously
-          await page.waitForVisible('.ace_editor.ace-dawn', true);
+          await page.waitForVisible('.ace_editor.ace-cloud_editor', true);
```

Comment:
There is `findCodeEditor().findEditor()` test util which only has a match when code editor finished loading